android/engine/src/main/java/org/smartregister/fhircore/engine/di/FhirEngineModule.kt

```diff
@@ -42,7 +42,7 @@ import timber.log.Timber
  * Provide [FhirEngine] dependency in isolation so we can replace it with a fake dependency in test
  */
 @InstallIn(SingletonComponent::class)
-@Module(includes = [CoreModule::class])
+@Module
```

Comment:
`FhirEngineModule` previously relied on dependencies from the `CoreModule`, was this refactored?

Reply:
The includes may have been unnecessary

Reply:
Nice.